lib/rpc/server/callable.go

```diff
@@ -48,20 +49,28 @@ func (c *osCommand) exec(ctx context.Context, stream pb.OutgoingMessageStream, a
 
 	err := cmd.Start()
 	if err != nil {
-		return trace.Wrap(err, "failed to start %v", cmd.Path)
+		return trace.Wrap(err, "failed to start").AddField("path", cmd.Path)
 	}
 
 	err = stream.Send(&pb.Message{Element: &pb.Message_ExecStarted{ExecStarted: &pb.ExecStarted{
 		Args: args,
 		Seq:  seq,
 	}}})
 	if err != nil {
-		// Do not wrap gRPC-specific error
-		return err
+		log.WithError(err).Warn("Failed to notify stream of command start.")
 	}
 	err = cmd.Wait()
 	if err == nil {
-		return stream.Send(&pb.Message{Element: &pb.Message_ExecCompleted{ExecCompleted: &pb.ExecCompleted{Seq: seq}}})
+		if err := stream.Send(&pb.Message{
```

Comment:
Nit: I would wrap stream.Send and error logging into a "void" function, would make the 3 places where you now just log the error instead of returning it a bit cleaner.

```go
func sendMessage(stream XXX, message *pb.Message) {
  err := stream.Send(message)
  if err != nil {
    log.WithError(err).Warn("Failed to send message %T.", message)
  }
}
```